cmd/bosun/sched/check_test.go

```diff
@@ -258,6 +258,68 @@ Loop:
 	}
 }
 
+// TestCheckNotifyUnknownDefault tests the default unknownTemplate.
+func TestCheckNotifyUnknownDefault(t *testing.T) {
+	s := new(Schedule)
+	nc := make(chan string, 1)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
do we really need to test the full post functionality to make sure this works? What about the approach used at https://github.com/bosun-monitor/bosun/blob/default-unknown-template/cmd/bosun/sched/check_test.go#L45 ?